src/main/java/me/totalfreedom/totalfreedommod/command/Command_seed.java

```diff
@@ -0,0 +1,93 @@
+package me.totalfreedom.totalfreedommod.command;
+
+import me.totalfreedom.totalfreedommod.rank.Rank;
+import net.md_5.bungee.api.chat.*;
+import net.md_5.bungee.api.chat.hover.content.Text;
+import org.bukkit.ChatColor;
+import org.bukkit.World;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@CommandPermissions(level = Rank.NON_OP, source = SourceType.BOTH)
+@CommandParameters(description = "Get the seed of the world you are currently in.", usage = "/seed [world]")
+public class Command_seed extends FreedomCommand
+{
+    @Override
+    public boolean run(CommandSender sender, Player playerSender, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
+    {
+        World world;
+
+        if (args.length > 0)
+        {
+            world = server.getWorld(args[0]);
+            if (world == null)
+            {
+                msg("That world could not be found", ChatColor.RED);
+                return true;
+            }
+        }
+        else
+        {
+            // If the sender is a Player, use that world. Otherwise, use the overworld as a fallback.
+            if (sender instanceof Player)
+            {
+                world = playerSender.getWorld();
+            }
+            else
+            {
+                world = server.getWorlds().get(0);
+            }
+        }
+
+        // If the sender is not a Player, use the usual msg method to
+        if (isConsole())
```

Comment:
The `senderIsConsole` variable is available to be used.

Reply:
Fixed